### DISCUSSION TITLE
Discussions moved to new project

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#60] [Announcements]
@Laravel-Lang/laravel-lang, since we now have a common project, I moved all the discussions from the Lang project to Common, because I thought it would be right to keep all the discussions in one place.

I will also release updated documentation in the near future.

**Comment by mohamedsabil83** [comment]
I agree with that. common will be the center of the all lang packages

**Comment by andrey-helldar** [comment]
Hi @caouecs!
 
Can you change the [list of popular](https://github.com/Laravel-Lang) repositories on the tile on the main page of the project?
 
I don't have the right to do this.
 
I offer this option:

``` 
common     - lang
attributes - http-statuses
publisher  - json-fallback-hotfix
```

**Reply by andrey-helldar** [reply]
Partially done: https://laravel-lang.com

Tomorrow I will add the statuses of translations from other projects 😎

**Reply by caouecs** [reply]
Is it possible to update the list of popular ? How ?

**Reply by andrey-helldar** [reply]
@caouecs, for example:

![Untitled Project](https://user-images.githubusercontent.com/10347617/219973698-2d8b60bd-94b8-47ed-8482-32d9f7cdfe69.gif)

**Reply by andrey-helldar** [reply]
I can't change(

![image](https://user-images.githubusercontent.com/10347617/219973760-f6c38d39-e2b8-42d4-9ee5-7e75ec154391.png)

**Reply by caouecs** [reply]
ok but it's pinned and not popular, no ? I try

**Reply by caouecs** [reply]
I found :)

**Reply by caouecs** [reply]
done

**Reply by andrey-helldar** [reply]
@caouecs, Wow, I just saw that these were really popular repositories :)

Just before, when this board was just added, they could be manually assigned :)

**Reply by andrey-helldar** [reply]
PS: The list is missing a few:

`lang`, `attributes` and `json-fallback-hotfix`

**Reply by caouecs** [reply]
done

**Reply by andrey-helldar** [reply]
Thank you

**Comment by ariaieboy** [comment]
@andrey-helldar can you link the Referents' names to their GitHub accounts in the documentation?

**Reply by andrey-helldar** [reply]
Yes, this is possible in two ways:

1. Manually indicate links using markdown marking.
2. Write a plugin for VuePress, which will automatically create them when compilation of documentation from Markdown files.

The first way is simple, but I'm too lazy. Therefore, a few months ago I began to create a plugin, but did not do it.
